modules/gds_printers/manifests/init.pp

```diff
@@ -0,0 +1,20 @@
+# Configure printers for Aviation House.
+#
+# You will need to override `ldap_username` if your LDAP username differs
+# from your local login username.
```

Comment:
I think that's fine then. They are the same for me, because my local login doesn't match LDAP.